lib/crypto-primitives/src/Cryptography/KDF/PBKDF2.hs

```diff
@@ -1,13 +1,140 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+
 module Cryptography.KDF.PBKDF2
     ( Parameters (..)
     , fastPBKDF2_SHA512
-    , generate
-    , prfHMAC
+
+    , generateKey
+    , generateKeyByron
+    , generateKeyIcarus
+    , generateKeyMetadata
```

Comment:
```suggestion
```

I would like to keep the specification of the numerical values in `Cardano.*`. See other comment.